src/python/T0/RunConfig/Tier0Config.py

```diff
@@ -24,6 +24,8 @@
 | |       |
 | |       |--> InjectMaxRun - Highest run number to be injected into the system from the SM database (default is unlimited)
 | |       |
+| |       |--> SpecifiedStreamNames - Streamer names to be ran exclusivley(default is None)
```

Comment:
Thanks for adding documentation. More than 'default is none', it would be better to show that `None` means that **all** streamers will be processed.